### ISSUE TITLE
Autolathe material costs need to be updated

### ISSUE THREAD

**Issue by ghost** [fortune13-ss13/thewasteland#546]
## Reproduction:
Go to an autolathe
Print a bonesetter
Shove it back in
Profit

Everyone knows this exploit, many people use it. It's an issue for almost all the items, though the bonesetter is one of the most noteworthy.

**Comment by Serxule** [comment]
i think this is with any item you can recycle per say, i inserted a bulletproof shield in the destructive lathe (to make a new one since it was damaged), and it ended up being worth a few shields in materials

**Comment by CocaColaTastesGood** [comment]
> i think this is with any item you can recycle per say, i inserted a bulletproof shield in the destructive analyzer (to make a new one since it was damaged), and it ended up being worth a few shields in materials, the destructive analyzer also is whats used for duping materials

Yeah, in general most returns on materials from recycling are way too high.
